client/src/pages/Home.jsx

```
@@ -127,13 +127,19 @@ export default function Home() {
-            <PaginationButton clickHandler={clickRightHandler}></PaginationButton>
+            {reviewCardLists?.length === 0 ? (
+              <div className={styles.noContentWrapper}>가장 첫번째 리뷰어가 되어보세요!</div>
+            ) : (
+              <>
+                <PaginationButton clickHandler={clickLeftHandler} side="left"></PaginationButton>
+                <ReviewCardLists
+                  reviewCardLists={reviewCardLists}
+                  type="main"
+                  size="big"
+                ></ReviewCardLists>
+                <PaginationButton clickHandler={clickRightHandler}></PaginationButton>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)